src/components/ViewRoutingSlip/AddManualTransactionDetails.vue

```diff
@@ -1,132 +1,114 @@
-            required
-            :rules="quantityRules"
-            v-model="transaction.quantity"
-            type="number"
-          >
-          </v-text-field>
-        </v-col>
-        <v-col cols="5">
-          <v-text-field
-            filled
-            label="Incorporation/Reference Number"
-            persistent-hint
-            :data-test="getIndexedTag('txt-incorporation', index)"
-            required
-            :rules="referenceNumberRules"
-            v-model.trim="transaction.referenceNumber"
-          >
-          </v-text-field>
-        </v-col>
-        <v-col cols="5" class="amount">
-          <v-text-field
-            filled
-            label="$Amount"
-            persistent-hint
-            :data-test="getIndexedTag('txt-amount', index)"
-            required
-            disabled
-            v-model="transaction.amount"
-          >
-          </v-text-field>
-          <div class="close-icon">
-            <v-btn
-              icon
-              class="mt-3 ml-1"
-              @click="removeManualTransactionRow(index)"
-              v-if="index > 0"
-              :data-test="getIndexedTag('btn-remove', index)"
-            >
-              <v-icon>mdi-close</v-icon>
-            </v-btn>
-          </div>
-        </v-col>
-
-        <v-col cols="2">
-          <v-checkbox
-            class="ma-0"
-            label="Priority Fee"
-            v-model="transaction.isPriorityFee"
-            hide-details
-            :data-test="getIndexedTag('check-priority', index)"
-          ></v-checkbox>
-        </v-col>
-        <v-col cols="8">
-          <v-checkbox
-            class="ma-0"
-            label="Future Effective Filing Fee"
-            v-model="transaction.isFutureEffectiveFiling"
-            hide-details
-            :data-test="getIndexedTag('check-future-effective', index)"
-          ></v-checkbox>
-        </v-col>
-        <v-col cols="12" class="mr-8">
-          <v-divider class="mt-4 mb-4" v-if="isDividerVisible(index)" />
-        </v-col>
-      </v-row>
-    </div>
-  </v-form>
+  <v-row dense class="mr-8" v-if="manualTransactionDetails">
+    <v-col cols="12">
+      <!-- calling @input rather than @keyup because @keyup used by the compoennt already !-->
+      <filing-type-auto-complete
+        v-model="manualTransactionDetails.filingType"
+        required
+        :rules="requiredFieldRule"
+        @input="delayedCalculateTotal()"
+      />
+    </v-col>
+    <v-col cols="2">
+      <v-text-field
+        filled
+        label="Quantity"
+        persistent-hint
+        :data-test="getIndexedTag('txt-quantity', index)"
+        required
+        :rules="requiredFieldRule"
+        v-model="manualTransactionDetails.quantity"
+        type="number"
+        @keyup="delayedCalculateTotal()"
+      >
+      </v-text-field>
+    </v-col>
+    <v-col cols="5">
+      <v-text-field
+        filled
+        label="Incorporation/Reference Number"
```

Comment:
i think referenceNumber is not mandatory rite?

Reply:
no. It is required 
![image](https://user-images.githubusercontent.com/78374865/132058579-4a895926-b0ad-4a7d-84ab-77a6636878d1.png)